content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,76 @@
+**Added** a Grafana dashboard for Citadel monitoring.
+**Added** [metrics](/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+**Added** control plane metrics to monitor Istio configuration state.
+**Added** telemetry reporting for traffic destined to `Passthrough` and `BlackHole` cluster.
+**Added** alpha support for in-proxy generation of service metrics using Prometheus.
+**Added** alpha support for environmental metadata in Envoy node metadata.
+**Added** alpha support for Proxy Metadata Exchange.
+**Added** alpha support for OpenCensus trace driver.
+**Improved** the mesh dashboard to provide monitoring of Istio configuration state.
+**Improved** the Pilot dashboard to expose additional key metrics to more clearly identify errors.
+**Removed** deprecated Adapter and Template custom resource definitions.
+**Deprecated** Mixer adapters. Extensions are expected to begin transition to new extension model. Legacy Mixer integration support will be maintained for 1.3 and 1.4.
+**Deprecated** Deprecated HTTP API Spec. The support for producing API attributes will be be removed in Istio 1.4.
+
+## Policy
+
+**Improved** rate limit enforcement to allow communication when the quota backend is unavailable.
+
+## Configuration Management
+
+**Added** the ability to configure Galley to use newer configuration processing.
+**Fixed** an issue in Galley where too many gRPC pings close connections.
+**Improved** Galley to avoid control plane upgrade failures.
+
+
+## istioctl
+
+**Added** many new istioctl experimental subcommands.
+- istioctl kube-uninject -f <descriptor.yaml> The inverse of kube-inject; removes Istio configuration from a Deployment descriptor.
+- istioctl describe pod <pod> Produce description of the Istio configuration affecting a pod
+- istioctl manifest apply Generates and applies Istio install manifest
+- istioctl manifest diff Show the the difference between two Istio manifests
+- istioctl manifest generate Generate Istio install manifest
+- istioctl manifest migrate Convert Helm install values to IstioControlPlane format
+- istioctl manifest versions List supported Istio versions
+- istioctl profile diff Show the difference between two Istio profiles
+- istioctl profile dump Show an Istio configuration profile
+- istioctl profile list Lists Istio configurations
+- istioctl add-to-mesh add local Kubernetes service or remote services(eg: services on VM) into Istio service mesh
+- istioctl remove-from-mesh remove local Kubernetes service or remote services(eg: services on VM) from Istio service
+**Improved** the istioctl experimental subcommands metrics, convert-ingress, and dashboard such that they are now supported Istio features.
+
+## Other
+
+**Added** New images based on distroless base images.
+**Improved** the Istio CNI Helm chart to have consistent versions with Istio.
+**Added** the ability for Kubernetes Jobs to exit correctly when finished.
```

Comment:
This is not true.... `Implemented a /quitquitquit endpoint in pilot-agent status server, which allows Kubernetes Jobs to notify the sidecar to exit once a job is complete.` was very intentionally worded. Kubernetes jobs will NOT exit when finished. Instead, we added a quit endpoint which ALLOWS jobs to exit once a job is complete IF they call it

Reply:
> This is not true.... `Implemented a /quitquitquit endpoint in pilot-agent status server, which allows Kubernetes Jobs to notify the sidecar to exit once a job is complete.` was very intentionally worded. Kubernetes jobs will NOT exit when finished. Instead, we added a quit endpoint which ALLOWS jobs to exit once a job is complete IF they call it. Not sure the best way to phrase this though?

please take a look at the latest text and see if it is sufficient to resolve this conversation.